docs/Reference/Pantheon-API-Methods.md

```diff
@@ -2703,6 +2703,163 @@ Reruns the transaction with the same state as when the transaction was executed.
     }
     ```
 
+### debug_traceBlock
+
+Returns full stack trace of all invoked opcodes of all transactions included in the block.
```

Comment:
```suggestion
Returns full trace of all invoked opcodes of all transactions included in the block.
```

While the trace can contain the stack after each opcode, it's not a stack trace as such.

Reply:
Fixed